### PR TITLE
DO-219: Multiple UX issue fixed.

### PR DESCRIPTION
**Pull request by amitjain24524**

### CODE REVIEW

File: src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx [hunk 1]
```diff
@@ -129,7 +129,7 @@ class MemberDetails extends React.Component {
                             <p>{removeLastString(personalDetails)}</p>
                           </div>
                           <div className="about">
-                            {otherDetails.aboutYou ? <p>otherDetails.aboutYou</p> : ''}
+                            {<p> { otherDetails.aboutYou } </p> || ''}
```
Comment (ayushmalik2110) [inline src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx:132]: use paragraph typography component.

File: src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileView/MemberProfileView.jsx [hunk 2]
```diff
@@ -153,7 +153,7 @@ class MemberProfileView extends React.Component {
                               <p>{removeLastString(personalDetails)}</p>
                             </div>
                             <div className="about">
-                              {otherDetails.aboutYou ? <p>otherDetails.aboutYou</p> : ''}
+                              {<p> { otherDetails.aboutYou } </p> || ''}
```
Comment (ayushmalik2110) [inline src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileView/MemberProfileView.jsx:156]: use paragraph typography component.

Comment (amitjain24524) [reply src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileView/MemberProfileView.jsx]: done

File: src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx (hunk 1 above)
Comment (amitjain24524) [reply src/components/routes/Member/AuthenticatedMember/MemberDetailsViewRoute/MemberDetailsViewRoute.jsx]: done

**Pull request merged**